superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -213,14 +215,19 @@ function ChartList(props: ChartListProps) {
   const canExport =
     hasPerm('can_export') && isFeatureEnabled(FeatureFlag.VERSIONED_EXPORT);
   const initialSort = [{ id: 'changed_on_delta_humanized', desc: true }];
-
+  const enableBroadUserAccess =
+    bootstrapData?.common?.conf?.ENABLE_BROAD_ACTIVITY_ACCESS || false;
```

Comment:
Probably better safe than sorry, but in a similar context on the backend we always do `config["ENABLE_BROAD_ACTIVITY_ACCESS"]` instead of ``config.get("ENABLE_BROAD_ACTIVITY_ACCESS")`, since we assume the variable should always be defined. So maybe we could get by with
```suggestion
    bootstrapData?.common?.conf?.ENABLE_BROAD_ACTIVITY_ACCESS;
```
(arguably that whole `bootstrapData` object with all nested fields should also be fully available)